spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
+  describe '#add' do
+    subject(:game) { described_class.new }
+
+    it 'adds the coin to the bottom of the column' do
+      column = 3
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[5][column]
+      expect(expected).to eql symbol
+    end
+    it 'adds the second coin on top of last one' do
+      column = 3
+      game.add(column)
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[4][column]
+      expect(expected).to eql symbol
+    end
+    it "doesn't do anything if the column is full" do
+      positions = Array.new(6) { Array.new([nil, nil, nil, 'X', nil, nil, nil]) }
+      column = 3
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      expected = game.board.positions
+      expect(expected).to eql positions
+    end
+  end
+
+  describe '#change_player' do
+    subject(:game) { described_class.new }
+    it 'changes active player' do
```

Comment:
This doesn't test what you say it tests, until you also assert what the value of `game.active_player` is before the call to `game.change_player`. Or even better, 
```
  expect { game.change_player }.to change(game.active_player).from("Player 1).to("Player 2")
```
See: https://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers:change